### DISCUSSION TITLE
Permalink里title是不是和name一样了？

### DISCUSSION THREAD

**Discussion by lephones** [hexojs/hexo#5266] [Question / Answer] [answered]
比如我的路径是source/_posts/my/hello.md
那:title生成的是my-hello
但是:name生成的也是my-hello
按官方说法 :name应该是hello

我看了一下filter下的permalink.js，是这里的代码处理的吧，/basename应该是只处理文件分割符'/'
```
    title: slug,
    name: typeof slug === 'string' ? basename(slug) : '',
```
附上官方介绍:
:title | 文件名称 (相对于 “source/_posts/“ 文件夹)
-- | --
:name | 文件名称

**Comment by uiolee** [comment, marked as answer]
`:title` is `my/hello`
`:name` is `hello`

i test in v6.3 and v7-rc2

### :title

```yml
permalink: /davfe/:title.html
```
```sh
INFO  Generated: davfe/my/hello.html
```

### :name

```yml
permalink: /davfe/:name.html
```
```sh
INFO  Generated: davfe/hello.html
```

**Reply by uiolee** [reply]
test in v6.3 and v7-rc2
